static/app/components/modals/commandPalette.spec.jsx

```diff
@@ -68,56 +74,38 @@ describe('Command Palette Modal', function () {
   });
 
   it('can open command palette modal and search', async function () {
-    const wrapper = mountWithTheme(
-      <App params={{orgId: 'org-slug'}}>{<div>placeholder content</div>}</App>,
-      TestStubs.routerContext([
+    render(<App params={{orgId: 'org-slug'}}>{<div>placeholder content</div>}</App>, {
+      context: TestStubs.routerContext([
         {
           router: TestStubs.router({
             params: {orgId: 'org-slug'},
           }),
         },
-      ])
-    );
-
-    // No Modal
-    expect(wrapper.find('Modal')).toHaveLength(0);
+      ]),
+    });
 
+    expect(screen.queryByRole('dialog')).not.toBeInTheDocument();
     openCommandPalette({params: {orgId: 'org-slug'}});
-    await tick();
-    await tick();
-    wrapper.update();
+    await waitFor(() => expect(screen.getByRole('dialog')).toBeInTheDocument());
```

Comment:
I think you can use await findByRole instead of the waitFor

Reply:
I considered that, but seems less good since I'm not using the result 🤔 is this is a stylistic preference, or is there a functional reason, too?

Reply:
yes `findByRole` would be preferable here according to the docs. Even if you are not using the result